.github/workflows/build-test.yml

```diff
@@ -45,7 +45,7 @@ env:
   DISABLE_INSIDERS_EXTENSION: 1 # Disable prompts to install Insiders in tests (else it blocks activation of extension).
   VSC_JUPYTER_INSTRUMENT_CODE_FOR_COVERAGE: true
   VSIX_NAME_PYTHON: 'ms-python-insiders.vsix'
-  VSC_JUPTYER_PYTHON_EXTENSION_VERSION: 'insiders'
+  VSC_JUPTYER_PYTHON_EXTENSION_VERSION: 'stable'
```

Comment:
Will revert (added for testing of another issue)

Reply:
Why revert? It makes tests pass for everybody. We can revert once Python fixes their issue, no?